api/hasura/cron/epochs.ts

```diff
@@ -272,7 +279,6 @@ export async function notifyEpochStart({
         },
         sanitize: false,
       });
-      updateEpochStartNotification(epoch.id);
```

Comment:
rather than doing this after reach notiifcation, I'm just doing it always at the bottom with the start activity.